hydra-node/test/Hydra/NetworkSpec.hs

```diff
@@ -40,8 +41,9 @@ spec = describe "Networking layer" $ do
     it "broadcasts messages to single connected peer" $ do
       received <- newEmptyMVar
       showLogsOnFailure $ \tracer -> failAfter 10 $ do
-        withOuroborosNetwork tracer (lo, 45778) [(lo, 45779)] (const @_ @(HydraMessage SimpleTx) $ pure ()) $ \hn1 ->
-          withOuroborosNetwork @(HydraMessage SimpleTx) tracer (lo, 45779) [(lo, 45778)] (putMVar received) $ \_ -> do
+        [port1, port2] <- fmap fromIntegral <$> randomUnusedTCPPorts 2
```

Comment:
:clap:

Reply:
Little remark maybe, the `fmap fromIntegral` is for converting from a Port type to Int or vice-versa? Would it make sense to align which ever side is using the untyped version?

Reply:
In this case, it's to convert from `Int` to `PortNumber`. I think it's better to have `randomUnusedTCPPorts` produce `PortNumber`. WDYT?

Reply:
Funnily, there's already a `fromIntegral` in the definition of `randomUnusedTCPPorts` 🙄

Reply:
>  I think it's better to have randomUnusedTCPPorts produce PortNumber. WDYT?

Yup!